### ISSUE TITLE
Working directly with .Rmd files

### ISSUE THREAD

**Issue by QuantScientist** [clayh53/tufte-jekyll#47]
Hello :) 
Is there any option to work directly with an .Rmd file? 

Thanks,

**Comment by ekstroem** [comment]
Rmd files work for me

* Write your stuff as an Rmd file. 
* Use the servr package. 
* Enjoy

**Comment by QuantScientist** [comment]
Hi,
They work for me too, though I did this manually, and it has **nothing to do with tufte-jekyll,**. 
It is just an Rmd rendered as an HTML page, see http://quantscientist.github.io/inferno/articles/16/ma1-process. But the look and feel is different from the regular tufte-jekill, e,g, http://quantscientist.github.io/inferno/ and I don't have any of the menu items since as recidted above, the HTML page was generated with server::

**Comment by ekstroem** [comment]
Yeah - I have the same issue. The two outputs are not .... quite identical. I think it may be a font issue or something although I haven't been able to figure out why I get different results. My main issue with rmd files is that I cannot use $$ $$ for displaymath (I know it is an abomination but old habits die hard)

**Comment by ekstroem** [comment]
If anyone has any clue with the the possible reason for the font difference I'd love to help trying to track this down. The feel and quality of the of Slomos Rmd-parsed html slide is a wee bit better
